### PR TITLE
SEP-0042 Stellar Asset Lists Standard

### PR DESCRIPTION
**Pull request by orbitlens**

### CODE REVIEW

File: ecosystem/sep-0042.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+- Community-managed Github repository for the [Asset Lists Catalogue]( https://github.com/stellar-asset-lists/index).  
+  It contains references to asset lists created by ecosystem providers. Client applications can use it to discover,
+  fetch available lists, and allow their users to select trusted providers in the interface. Web UI for this catalogue
+  is available [here](https://stellar.expert/asset-lists).
+- Basic [JS library](https://www.npmjs.com/package/@stellar-asset-lists/sdk) that simplifies integration of asset lists
+  into web applications.
+- [StellarExpert Top50](https://api.stellar.expert/explorer/public/asset-list/top50) asset list for Stellar Pubnet.
+- [StellarExpert Testnet Top50](https://api.stellar.expert/explorer/testnet/asset-list/top50) asset list for Stellar
+  Testnet.
+
```
Comment (leighmcculloch) [inline ecosystem/sep-0042.md:218]: Can we add a changelog at the bottom, from the [sep-template.md](https://github.com/stellar/stellar-protocol/master/sep-template.md)?
```suggestion
## Changelog

- `v0.2.0`: Initial draft. [#1440](https://github.com/stellar/stellar-protocol/pull/1440)
```

**Review by leighmcculloch — COMMENTED** [review]
Could you add the changelog section to set us up for keeping that updated, otherwise looks good to merge.

**Comment by silence48** [comment]
> This pull request is stale because it has been open for 30 days with no activity. It will be closed in 30 days unless the stale label is removed.

@orbitlens  @leighmcculloch I had been away for a bit, but am back now... this PR should not go stale, maybe you can remove the label.  What do we still need to do in order to implement this? What is the status of the mailing list discussion?

Thanks!

**Review by leighmcculloch — APPROVED** [review]

**Comment by orbitlens** [comment]
Thank you, Leigh

**Pull request merged**